04-ruby-mvc-sinatra/ruby-basics/readme.md

```diff
@@ -327,7 +327,10 @@ b.upcase!
 a
 # => "CHEESEBURGER"
```

Comment:
are we leaving these examples as is? or should we consider staying consistent with "dog", etc.